src/vibe_llama/scaffold/samples/human_in_the_loop.py

```diff
@@ -0,0 +1,83 @@
+HUMAN_IN_THE_LOOP_CODE = """
```

Comment:
I feel like we should be picky and curate these so that they are high quality for our ideal customer profile. 2 joke flows seems like a bit much.